src/overlays/actors/ovl_En_Elfgrp/z_en_elfgrp.c

```diff
@@ -61,14 +61,14 @@ void EnElfgrp_Init(Actor* thisx, GlobalContext* globalCtx) {
     EnElfgrp* this = THIS;
     s32 sp24;
 
-    this->unk_147 = ENELFGRP_GET(&this->actor);
+    this->unk_147 = STRAY_FAIRY_TYPE(&this->actor);
     this->unk_148 = 0;
     this->unk_14A = 0;
     this->actor.focus.pos.y += 40.0f;
     this->actor.flags &= ~ACTOR_FLAG_1;
 
     switch (this->unk_147) {
-        case ENELFGRP_1:
+        case STRAY_FAIRY_TYPE_FAIRY_FOUNTAIN:
```

Comment:
Same thing here; you want to use `ENELFGRP_1`, since this is checking the result of `ENELFGRP_GET`
```suggestion
        case ENELFGRP_1:
```